build_iso.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-wd=`dirname $0`
+wd=${dirname $0}
```

Comment:
Why change this?
And {} is incorrect wd=$(dirname $@)

Reply:
change was recommended by shellcheck : https://www.shellcheck.net/wiki/SC2006